client/login/wp-login/test/test.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External Dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal Dependencies
+ */
+import { Login } from '../index.jsx';
+import useFakeDom from 'test/helpers/use-fake-dom';
+
+const defaultProps = {
+	magicLoginEnabled: true,
+	magicLoginView: '',
+	translate: ( str ) => {
```

Comment:
that's one of the dependencies, so moking it makes sense to me, you can do `translate: str => str,`